### ISSUE TITLE
scx_bpfland: Support ignoring CPUFreq EPP for systems that don't support it

### ISSUE THREAD

**Issue by anh0516** [sched-ext/scx#560]
On CPUs that are too old to support pstate EPP, or on kernels built without CPUFreq support, scx_bpfland repeatedly tries to read the energy_performance_preference, but it doesn't exist.

Here's an strace snippet:
```
openat(AT_FDCWD, "/sys/devices/system/cpu/cpufreq/policy0/energy_performance_preference", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
sched_yield()                           = 0
sched_yield()                           = 0
sched_yield()                           = 0
sched_yield()                           = 0
sched_yield()                           = 0
sched_yield()                           = 0
sched_yield()                           = 0
sched_yield()                           = 0
futex(0x60d31239cb40, FUTEX_WAIT_BITSET_PRIVATE, 4294967295, {tv_sec=757, tv_nsec=435783311}, FUTEX_BITSET_MATCH_ANY) = -1 ETIMEDOUT (Connection timed out)
sched_yield()                           = 0
sched_yield()                           = 0
sched_yield()                           = 0
sched_yield()                           = 0
openat(AT_FDCWD, "/sys/devices/system/cpu/cpufreq/policy0/energy_performance_preference", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
sched_yield()                           = 0
sched_yield()                           = 0
sched_yield()                           = 0
sched_yield()                           = 0
sched_yield()                           = 0
sched_yield()                           = 0
sched_yield()                           = 0
sched_yield()                           = 0
futex(0x60d31239cb40, FUTEX_WAIT_BITSET_PRIVATE, 4294967295, {tv_sec=758, tv_nsec=436022196}, FUTEX_BITSET_MATCH_ANY) = -1 ETIMEDOUT (Connection timed out)
sched_yield()                           = 0
sched_yield()                           = 0
sched_yield()                           = 0
sched_yield()                           = 0
openat(AT_FDCWD, "/sys/devices/system/cpu/cpufreq/policy0/energy_performance_preference", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
sched_yield()                           = 0
sched_yield()                           = 0
sched_yield()                           = 0
sched_yield()                           = 0
sched_yield()                           = 0
sched_yield()                           = 0
sched_yield()                           = 0
sched_yield()                           = 0
futex(0x60d31239cb40, FUTEX_WAIT_BITSET_PRIVATE, 4294967295, {tv_sec=759, tv_nsec=436235040}, FUTEX_BITSET_MATCH_ANY) = -1 ETIMEDOUT (Connection timed out)
```

I'm not sure what the `futex() ETIMEDOUT` is about; if it's related to this, or if it's an artifact of my kernel configuration.

A solution would be to implement the same `--no-freq-scaling` flag used by scx_lavd.

**Comment by arighi** [comment]
It should only trigger an openat() every second, it's not a lot of overhead. But I can definitely add a option to prevent doing so. Or maybe I can just check if the file exists during init and, if it's not there, automatically disable this periodic check...

**Comment by arighi** [comment]
Actually #561 should solve this easily...

**Comment by anh0516** [comment]
Oh for sure that's even simpler.

**Comment by anh0516** [comment]
Fixed by #561.

Do you also get the `futex()` error with Linux 6.10.6, or is that something I should debug on my own system, what with my kernel configuration and the other patches applied?
